products/umbraco.md

```diff
@@ -69,7 +69,7 @@ releases:
 
 -   releaseCycle: "6"
     # https://umbraco.com/blog/umbraco-600-released/
-    releaseDate: 2013-01-31
+    releaseDate: 2013-03-27
```

Comment:
Wrong update, but 6.0.0 tag does not exist (https://github.com/umbraco/Umbraco-CMS/releases/tag/release-6.0.0), so cannot be avoided.